tests/test_hawk_http_backend.py

```diff
@@ -0,0 +1,45 @@
+from cachish.backends import Hawk
+
+import responses
+
+@responses.activate
+def test_hawk_backend():
+    response = {
+        "key1": "value1",
+        "key2": "value2",
+    }
+    responses.add(responses.GET, 'https://example.com',
+        json=response)
+    config = {
+        'secret_id': 'supersecret',
```

Comment:
The secret_id is not actually secret, it's sent in the clear as part of the header, so this should probably be renamed to reflect that.